lib/elixir/lib/kernel.ex

```diff
@@ -932,8 +942,11 @@ defmodule Kernel do
       iex> 1 / 2
       0.5
 
-      iex> 2 / 1
-      2.0
+      iex> -3.0 / 2.0
+      -1.5
+
+      iex> 5 / 1
+      5.0
```

Comment:
I don't really see the point in changing `2 / 1` to `5 / 1`. Same for the trunc examples above.